slides/developer-guide/working-with-shapes/adding-shapes-to-a-powerpoint-presentation/_index.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Adding-Shapes-to-a-PowerPoint-Presentation"
+type: docs
+url: /adding-shapes-to-a-powerpoint-presentation/
+weight: 25
+---
+
+## **Introduction**
+
```

Comment:
refactor this sentence
_you should prepare an appropriate data transfer object and pass it to the parameters or specify an index of the shape to be cloned_
It's ambiguous between
_(you should prepare an appropriate data transfer object and pass it to the parameters) or (specify an index of the shape to be cloned)_
and
_(you should prepare an appropriate data transfer object) and (pass it to the parameters or specify an index of the shape to be cloned)_

I suggest mentioning clone feature in a separate sentence.